src/Countdown.elm

```diff
@@ -0,0 +1,103 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
```

Comment:
Don't expose unnecessary symbols - `Msg`'s constructors (`Msg(..)` -> `Msg`) and `main` shouldn't be used externally.